smg-automotive/components-pkg
# feat: added checkbox component

References
> [Jira](https://autoricardo.atlassian.net/browse/DM-152)
> [Checkbox component (figma)](https://www.figma.com/file/CHU4fpMfXpiAW4qNFzi9y1/AS24-WebComponents?node-id=4%3A5558)
> [Lead message (figma)](https://www.figma.com/file/f0LWQupJueW6KHBQujVGe4/Lead-Form-Page-(Working-Title)?node-id=2%3A58850)

## Motivation and context

Extracting Checkbox component. The invalid checkbox state is not implemented because is not integrated here [Lead message (figma)](https://www.figma.com/file/f0LWQupJueW6KHBQujVGe4/Lead-Form-Page-(Working-Title)?node-id=2%3A58850). We ONLY extract what is on the lead message. In any case let me know if it's better to introduce that state on this PR

## Before

No checkbox component😱

## After

Checkbox component 🥳

<img width="1098" alt="Screenshot 2022-07-06 at 14 00 51" src="https://user-images.githubusercontent.com/37380787/177545468-3798d657-8c84-4132-a6af-407de5620d59.png">

## How to test

Please check the checkbox states: selected, unselected, disabled, hover. And the design.
We don't have the branch deployment yet but you can check the branch locally.

## Thank you 🌮
